examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.c

```diff
@@ -0,0 +1,837 @@
+/*
```

Comment:
I am confused.  Don't you already have this in `examples/platform/silabs/SiWx917/SiWx917/rsi_if.c`

Reply:
We are not using ` rsi_if.c `and new file `sl_wifi_if.c` has been created while integrating with wifi-sdk.

Reply:
rsi_if.c is removed in latest commit